### PR TITLE
Dynamically Get Latest Chart Version From Repo

### PR DESCRIPTION
**Pull request by mphammer**
NOTE: Do not merge until we get approval from all teams on the package name convention

### CODE REVIEW

File: pkg/globals/helmglobalvalues.go [hunk 1]
```diff
@@ -31,40 +37,35 @@ var BaseChartRepository = "https://sig-repo.synopsys.com/sig-cloudnative"
...
-var OpsSightVersionToChartVersion = map[string]string{
-	"2.2.5": "2.2.5-1",
-}
-
 // OpsSightVersion ...
-var OpsSightVersion = "2.2.5"
+var OpsSightVersion = ""
 
 // OpsSightChartName ..
 var OpsSightChartName = "opssight"
```
Comment (msenmurugan) [inline pkg/globals/helmglobalvalues.go]: change them to blackduck-connector?

File: pkg/util/helm_helpers.go [hunk 2]
```diff
@@ -341,6 +353,183 @@ func LoadChart(chartURL string, actionConfig *action.Configuration) (*chart.Char
...
+
+	username := client.ChartPathOptions.Username
+	password := client.ChartPathOptions.Password
+	// chartName := chartURL
+	// chartVersion :=  strings.TrimSpace(client.ChartPathOptions.Version)
+	certFile := client.ChartPathOptions.CertFile
+	keyFile := client.ChartPathOptions.KeyFile
+	caFile := client.ChartPathOptions.CaFile
+
+	s := initSettings()
```
Comment (msenmurugan) [inline pkg/util/helm_helpers.go]: use settings variable instead

Comment (mphammer) [reply pkg/util/helm_helpers.go]: Done! :)

**Review by msenmurugan — APPROVED** [review]
/lgtm

**Pull request merged**